materialize-bigquery/bigquery.go

```diff
@@ -48,166 +44,184 @@ func (c *config) Validate() error {
 	return nil
 }
 
+func (c *config) client(ctx context.Context) (*client, error) {
+	var clientOpts []option.ClientOption
+
+	credentials, err := base64.StdEncoding.DecodeString(string(c.CredentialsJSON))
```

Comment:
Requiring base64-encoded credentials is not consistent with other google connectors we have and generally pretty annoying, but I left it alone for now to maintain backward compatibility. I think it is something that will need to be addressed eventually, and could probably even be done in a backward-compatible way but I have left that (and various other improvement tasks) for another day.